src/pages/Home/index.tsx

```
@@ -0,0 +1,255 @@
+  const [search, setSearch] = useState<string>("");
+  const [colorFilter, setColorFilter] = useState<string[]>([]);
+
+  const searchLower = search.trim().toLowerCase();
+
+  const [open, setOpen] = useState<boolean>(false);
+
+  const [editOpen, setEditOpen] = useState<boolean>(false);
+  const [noteToEdit, setNoteToEdit] = useState<INote | null>(null);
+  const [noteToRemove, setNoteToRemove] = useState<INote | null>(null);
+
+  const favoriteNotes = notes && notes.filter((note) => note.isFavorite);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)